packages/client-sdk-nodejs/src/internal/pubsub-client.ts

```diff
@@ -169,11 +169,13 @@ export class PubsubClient extends AbstractPubsubClient<ServiceError> {
       topic: options.topicName,
       resume_at_topic_sequence_number:
         options.subscriptionState.resumeAtTopicSequenceNumber,
+      sequence_page: options.subscriptionState.lastTopicSequencePage,
     });
 
     this.getLogger().trace(
-      'Subscribing to topic with resume_at_topic_sequence_number: %s',
-      options.subscriptionState.resumeAtTopicSequenceNumber
+      'Subscribing to topic with resume_at_topic_sequence_number %s and sequence_page %s',
+      options.subscriptionState.resumeAtTopicSequenceNumber,
+      options.subscriptionState.lastTopicSequencePage ?? 'undefined'
```

Comment:
See also below my comment about how `resumeAtTopicSequenceNumber` is implemented. We can do the same for `resumeAtTopicSequencePage` to avoid null checks.